sql/org_mozilla_fenix_nightly_derived/baseline_clients_last_seen_v1/query.sql

```diff
@@ -5,6 +5,9 @@ WITH _current AS (
     -- 28 days for each usage criterion as a single 64-bit integer. The
     -- rightmost bit represents whether the user was active in the current day.
     CAST(TRUE AS INT64) AS days_seen_bits,
+    udf.days_since_created_profile_as_28_bits(
+      DATE_DIFF(submission_date, first_run_date, DAY)
+    ) AS days_created_profile_bits,
```

Comment:
This is a detail I wanted to avoid, but it turns out it's necessary to maintain compatibility with the overall smoot table structure and how we handle desktop data.